exporter/tanzuobservabilityexporter/README.md

```diff
@@ -53,32 +46,36 @@ exporters:
 service:
   pipelines:
     traces:
-      receivers: [ examplereceiver ]
-      processors: [ batch ]
-      exporters: [ tanzuobservability ]
+      receivers: [examplereceiver]
+      processors: [batch]
+      exporters: [tanzuobservability]
 ```
 
 ### Advanced Configuration
 
-#### Processor
+#### Processors
 
 The memory limiter processor is used to prevent out of memory situations on the collector. It allows performing periodic
-checks of memory usage if it exceeds defined limits will begin dropping data and forcing GC to reduce memory
-consumption. [Details and defaults here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md).
+checks of memory usage – if it exceeds defined limits it will begin dropping data and forcing garbage collection to
+reduce memory
+consumption. [Details and defaults here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md)
+.
 
-#### Exporter
+#### Exporters
```

Comment:
I don't think this should be plural. There should only be a single exporter that we're documenting: the `tanzuobservability` exporter.